src/main/java/org/humancellatlas/ingest/core/MetadataDocument.java

```diff
@@ -35,7 +30,7 @@
     private Object content;
 
     @Indexed
-    private @DBRef Set<SubmissionEnvelope> submissionEnvelopes = new HashSet<>();
+    private @Setter @DBRef SubmissionEnvelope submissionEnvelope;
```

Comment:
As discussed offline, please don't forget to retain the link of submission envelopes inside the project for now!